Client/mods/deathmatch/logic/lua/CLuaFunctionParseHelpers.cpp

```diff
@@ -1093,6 +1093,30 @@ bool MinClientReqCheck(CScriptArgReader& argStream, const char* szVersionReq, co
     return true;
 }
 
+//
+// Check min client is correct
+// Thrown a error if below required
+//
+void MinClientReqCheck(lua_State* luaVM, const char* szVersionReq, const char* szReason)
+{
+    CLuaMain* pLuaMain = g_pClientGame->GetLuaManager()->GetVirtualMachine(luaVM);
+    if (pLuaMain)
+    {
+        CResource* pResource = pLuaMain->GetResource();
+        if (pResource)
+        {
+            if (pResource->GetMinClientReq() < szVersionReq)
+            {
+                #if MTASA_VERSION_TYPE == VERSION_TYPE_RELEASE
+                SString err("<min_mta_version> section in the meta.xml is incorrect or missing (expected at least %s %s because %s)", "client",
+                                      szVersionReq, szReason);
```

Comment:
Couldnt you just put `"client"` into the err itself?
```cpp
SString err("<min_mta_version> section in the meta.xml is incorrect or missing (expected at least client %s because %s)", szVersionReq, szReason);
```